### PR TITLE
added combined card-accordion component

### PR DESCRIPTION
**Pull request by kevinpapst**
## Description

Replaces #162 

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] I updated the documentation (see [here](https://github.com/kevinpapst/TablerBundle/tree/master/docs))
- [x] I agree that this code will be published under the [MIT license](https://github.com/kevinpapst/TablerBundle/blob/master/LICENSE)

### CODE REVIEW

File: templates/embeds/collapsible.html.twig [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{% extends '@Tabler/embeds/card.html.twig' %}
+{% set collapsed = false %}
```
Comment (cavasinf) [inline templates/embeds/collapsible.html.twig:2]: Is this still needed?

File: templates/embeds/collapsible.html.twig [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+{% set fullsize = true %}
```
Comment (cavasinf) [inline templates/embeds/collapsible.html.twig:3]: Parameter under `extends` works as untended?
I always put them above the extends , even more if the extend use it.

File: templates/embeds/collapsible.html.twig [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+{% block box_title %}{% endblock %}
+{% block box_body %}
+    {% set _id = id ?? tabler_unique_id('collapsible_') %}
+    {% set _open = open ?? false %}
+    {% set _id_collapse = _id ~ '_item' %}
```
Comment (cavasinf) [inline templates/embeds/collapsible.html.twig:8]: Can we put that on top?
(part of me want that 😄)

File: templates/embeds/collapsible.html.twig [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+                </button>
+            </div>
+
+            <div id="{{ _id_collapse }}" class="accordion-collapse collapse {% if _open %}show{% endif %}">
+                <div class="accordion-body {% block body_class %}pt-0{% endblock %}">
+                    {% block body %}{% endblock %}
+                </div>
+            </div>
+        </div>
+    </div>
```
Comment (cavasinf) [inline templates/embeds/collapsible.html.twig:26]: I understand that the `accordion` component is scary and not easy to manipulate/use.
But isn't it more "Maintainers" friendly to use it?
The less duplicate code we have to maintain, the happier we are?

**Comment by cavasinf** [comment]
Still using blocks `title` and `body`.
I'll go like that:
```twig
{% set fullsize = true %}
{% extends '@Tabler/embeds/card.html.twig' %}
{% from '@Tabler/components/accordion.html.twig' import accordion %}

{% block box_class %}border-0{% endblock %}
{% block box_title %}{% endblock %}

{% set _accordionItem = {
    title: block('title') ?? '',
    body: block('body') ?? '',
    bodyClasses: '',
    options: {
        open: false,
        bodyExtraClass: 'border-top',
    }
} | merge(accordionItem ?? {}) %}

{% set _accordionOption = {} | merge(accordionOption ?? {}) %}

{% block box_body %}
    {{ accordion(
        [_accordionItem],
        _accordionOption
    ) }}
{% endblock %}
```

Used like that:
````twig
{% set item = {
    options : {
        open : true
    }
} %}
{% embed '@theme/embeds/collapsible.html.twig' with {accordionItem  : item} %}
    {% block title %}File permissions{% endblock %}
    {% block body %}
        .....
    {% endblock %}
{% endembed %}
```

WDYT @kevinpapst?

**Comment by kevinpapst** [comment]
I'll give this a try later today ... full day of meetings.
Looks like a clever solution indeed, better than mine 😁

File: templates/embeds/collapsible.html.twig (hunk 2 above)
Comment (kevinpapst) [reply templates/embeds/collapsible.html.twig:3]: To me an extends belong to the top of the file, to understand the structure.
But yes, it works even below. If you look at the compiled class for this template, you'll understand why.

File: templates/embeds/collapsible.html.twig (hunk 1 above)
Comment (kevinpapst) [reply templates/embeds/collapsible.html.twig:2]: Yes, The should always be collapsed, and then you define through the accordionItem `open` flag if the content is visible or not.

**Comment by kevinpapst** [comment]
Small adjustments applied, to simplify passing options.

```
    {% embed '@theme/embeds/collapsible.html.twig' with { boxtype: 'danger', border: false, open: true, flush: true } %}
        {% block title %}Foo{% endblock %}
        {% block body %}Bar{% endblock %}
```

Can be reviewed @cavasinf

**Pull request merged**